bower.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "eva-web",
+  "version": "1.0.0",
```

Comment:
Should be 2.1, because 2.0 was tagged for PAG, and this is a retro-compatible improvement.